examples/service-catalog/service-catalog.yaml

```diff
@@ -49,17 +49,19 @@ objects:
     verbs:
     - create
 
-- kind: ClusterRoleBinding
-  apiVersion: v1
+- apiVersion: authorization.openshift.io/v1
+  kind: ClusterRoleBinding
   metadata:
     name: service-catalog-sar-creator-binding
   roleRef:
     name: sar-creator
-  userNames:
-    - system:serviceaccount:kube-service-catalog:service-catalog-apiserver
+  subjects:
+  - kind: ServiceAccount
+    name: service-catalog-apiserver
+    namespace: kube-service-catalog
```

Comment:
@pmorie I suggest choosing a canonical namespace and this one looks pretty good.  `kube-service-catalog` is different than `service-catalog` since we reserve `kube-*` and `openshift-*` to make sure normal users don't get ahold of them.